docs/essentials/backgrounds.mdx

```diff
@@ -5,136 +5,187 @@ sidebar:
 
 <Video src="../_assets/essentials/addon-backgrounds-optimized.mp4" />
 
 ## Configuration
 
 By default, the backgrounds toolbar includes a light and dark background.
 
-But you're not restricted to these backgrounds, you can configure your own set of colors with the `parameters.backgrounds` [parameter](../writing-stories/parameters.mdx) in your [`.storybook/preview.js`](../configure/index.mdx#configure-story-rendering):
+But you're not restricted to these backgrounds. You can configure your own set of colors with the `backgrounds` [parameter](../writing-stories/parameters.mdx) in your [`.storybook/preview.js`](../configure/index.mdx#configure-story-rendering). There are two ways to configure the backgrounds:
 
-{/* prettier-ignore-start */}
+### Using `options`
 
-<CodeSnippets path="storybook-preview-configure-background-colors.md" />
+<Callout variant="info">
+  This API can only be used when the [`backgroundsStoryGlobals` feature flag](../api/main-config/main-config-features.mdx#backgroundsstoryglobals) is enabled.
+</Callout>
 
-{/* prettier-ignore-end */}
+The available background colors are defined using the `options` property:
 
-If you define the `default` property, the addon will apply it to all stories. Otherwise, it's only listed as an available color.
+{/* prettier-ignore-start */}
 
-### Overriding backgrounds with globals
+<CodeSnippets path="addon-backgrounds-options-in-preview.md" />
 
-The Backgrounds addon enables you to change the background color applied to a story by selecting from the list of predefined background colors in the toolbar. If you need, you can opt into the globals API to configure backgrounds to ensure that a story always uses a specific background color regardless of the existing configuration.
+{/* prettier-ignore-end */}
 
-To enable it, you'll need to take some additional steps to set it up properly. Our recommendation for configuring and using the globals API with the Backgrounds addon is detailed below.
+### Using `values`
 
-Update your [`.storybook/main.js|ts`](../configure/index.mdx#configure-your-storybook-project) to include the [`backgroundsStoryGlobals`](../api/main-config/main-config-features.mdx#backgroundsstoryglobals) option within Storybok's `features` configuration.
+If you're not using the `globals` API, you can define the available background colors using the `values` property:
 
 {/* prettier-ignore-start */}
 
-<CodeSnippets path="main-config-features-backgrounds-story-globals.md" />
+<CodeSnippets path="addon-backgrounds-values-in-preview.md" />
 
 {/* prettier-ignore-end */}
 
-<Callout variant="info">
+With either method, available backgrounds can be defined at the project, component, and story levels.
 
-Enabling this API will turn off the ability to change any options from the toolbar to ensure that the background is always set to the intended value for the story.
+To apply a default background color, assign the `default` property to one of the available colors.
```

Comment:
@ndelangen — Is this API still available when using story globals? I could see value in being able to define a default that is still controllable via the toolbar. But I could also see that complicating the code...

Reply:
The code snippet is the old behavior, which only works if the feature flag is off.

The `default` key is ignored when the feature flag is true.